docker/dockerfiles/be/upload_coredump.sh

```diff
@@ -0,0 +1,70 @@
+#!/usr/bin/bash
+
+coredump_log()
+{
+    echo "[`date`] [coredump] $@" >&2
+}
+
+COREDUMP_PATH=/opt/starrocks/be/storage/coredumps
```

Comment:
use $STARROCKS_HOME, to be compatible with cn deployment as well.

Reply:
whether is this coredump path can be configured or read from `/proc/sys/kernel/core_pattern`?

Reply:
It could be a little issue if wishing to mount a separate volume specifically  for the coredump. This hardcode makes it difficult to separate them. And potentially ruin the BE, if the core takes all the space of be/storage/, BE diskspace check will fail and can't start.